File: translation_output/wizardcoder/codenet/java/python-sanitized/p03963.py
import java.util.*;

class p03963:
    def main(self):
        sc = Scanner(System.in)
        N = sc.nextInt()
        K = sc.nextInt()
        ans = K
        for i in range(2, N+1):
            ans *= K-1
        print(ans)

p03963().main()

File: Results/DER/Translation/translation_output/misleading_test/deepseekcoder/codenet/python/java-sanitized/p03963.java
import java.util.Scanner;

public class p03963 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        int k = scanner.nextInt();
        System.out.println(k * (int)Math.pow((k - 1), (n - 1)));
    }
}